dabl-meta.yaml

```diff
@@ -4,13 +4,14 @@
 catalog:
     name: dabl-integration-symphony
     group_id: com.digitalasset
-    version: 0.2.0
+    version: 0.2.1
     short_description: Symphony
-    description: Symphony Send and Receive
+    description: Send and Receive messages and form elements through the Symphony messaging app
```

Comment:
Does this fit on screen?

Reply:
Thinking out loud, it would be nice if we could include links in these descriptions...

Reply:
Yeah, I was wanting to be able to put in a link to Symphony's website a while ago when writing that blurb.

Here's the description in context (after changing "through" to "via"):
![Screen Shot 2021-02-04 at 10 24 06 AM](https://user-images.githubusercontent.com/71082197/106914575-28963f80-66d3-11eb-8b27-608af66538b5.png)

A bit longer than the others, but I think having "Symphony messaging app" is useful for those who might not know what Symphony is, vs. Slack which is ubiquitous.

Let me know what you think.